### PR TITLE
Add Osmium production

### PR DESCRIPTION
**Pull request by redcreeper14385**
The bits for this aren't in the mod yet, if @Jerome226 wants to then he can but I am happy to as well.
I might have screwed this up somewhere so let me know if that is the case.

### CODE REVIEW

**Comment by Jerome226** [comment]
Will add the items required into the mod before merging.

**Comment by redcreeper14385** [comment]
makes sense, just wanted to clear it as done :)

**Comment by redcreeper14385** [comment]
Converted to draft in the mean time.

File: kubejs/data/superpackutils/tags/infuse_types/nickel_selenide.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "replace": false,
+    "values": [
+        "superpackutils:nickel_selenide"
```
Comment (Jerome226) [inline kubejs/data/superpackutils/tags/infuse_types/nickel_selenide.json:4]: This is not required in the pack files, the tags are added by the mod itself.

**Pull request merged**